doc/Tex/RAM_SCB_install.tex

```diff
@@ -1,180 +1,72 @@
-encouraged to explore the websites and help files associated with each package.
-
-
-\subsubsection{Ncarg Quick Install}
-Installation of the NCAR Graphics library from source is an involved process.  
-It is strongly suggested that the user first check their system to see if the 
-library is already installed or install via binary.
-
-\subsubsection{NetCDF Quick Install}
-Unpack the NetCDF tarball, {\tt cd} into the unzipped directory, and use the 
-following commands:
-\begin{verbatim}
-./configure --prefix=/Users/ram_user/netcdf_lib
-make install
-\end{verbatim}
-This will install the NetCDF library in {\tt /Users/ram\_user/netcdf\_lib} 
-(the home directory for user \textit{ram\_user}); change the value of 
-{\tt --prefix} to select the installation destination.  For installation, 
-NetCDF requires only a C compiler.  {\tt gcc} is strongly suggested.
-
-\subsubsection{Pspline Quick Install}
-Pspline requires NetCDF, so be sure that NetCDF is correctly installed before 
-continuing.  After unpacking the tarball, compile the library using
-\begin{verbatim}
-make FORTRAN_VARIANT=Portland NETCDF_DIR=~/netcdf_lib
-\end{verbatim}
-It is important to specify the Fortran compiler that coincides with the 
-compiler you will use to make the RAM-SCB executable.  Other options accepted 
-by the {\tt FORTRAN\_VARIANT} variable are {\tt PathScale},  {\tt GCC}(gfortran
-and g95), {\tt NagWare},  {\tt Fujitsu}, {\tt Intel}, and {\tt Absoft}.  
-Ensure that the value of {\tt NETCDF\_DIR} coincides with the installation 
-directory of NetCDF.
-
-{\tt make} may fail during the creation of the Pspline test utilities.  If 
-this happens, check for the creation of a {\tt lib} directory and several 
-\verb*l*.al files contained within.  If they exist, disregard the error 
-message and continue to the next step.
-
-Install Pspline as follows (changing the value of {\tt PREFIX} to select 
-installation directory):
-\begin{verbatim}
-make install PREFIX=~/psline_portland
-\end{verbatim}
-On machines where you will use different compilers, it is helpful to add the 
-compiler name to the name of the installation directory.
+% QUESTION: All of the information in this section is out of date. Even the NETCDF installation instructions are incorrect given that they have split it into a C version and a Fortran version. This is still easy enough to describe and walk through, but the installation of GSL from source isn't as easy. Do we really want to have this section?
 
 \subsection{Libraries From Modules}
-Well maintained clusters often use the {\tt module} interface for loading
-and unloading software packages.  Be sure to explore the available packages
-on a new system before going through the arduous task of installing from source!
+Well maintained clusters often use the {\tt module} interface for loading and unloading software packages.  Be sure to explore the available packages on a new system before going through the arduous task of installing from source!
```

Comment:
Most places with a `module` system also have support... It's probably spoon-feeding, but maybe say that before installing from source it's a good idea to see if the cluster support can build modules for you.

Reply:
Hand holding and spoon feeding aren't necessarily bad things. If it will get more people to use the code.